app/assets/javascripts/govuk_publishing_components/components/checkboxes.js

```diff
@@ -23,6 +23,22 @@ window.GOVUK.Modules = window.GOVUK.Modules || {};
         }
       });
 
+      $(scope).find('input[type=checkbox]').on('change', function(e) {
+        var $checkbox = $(e.target);
+        var category = $checkbox.data("track-category");
+        if (typeof category !== "undefined") {
+          var isChecked = $checkbox.is(":checked");
+          var uncheckTrackCategory = $checkbox.data("uncheck-track-category");
+          if (!isChecked && typeof uncheckTrackCategory !== "undefined") {
+            category = uncheckTrackCategory;
+          }
+          var action = $checkbox.data("track-action");
+          var label = $checkbox.data("track-label");
+          var value = $checkbox.data("track-value");
```

Comment:
What is `track-value`? And do we need to also be fetching `track_options` if they are passed into the component?